privaterelay/settings.py

```diff
@@ -242,6 +243,7 @@ def _get_initial_middleware():
 SUBSCRIPTIONS_WITH_UNLIMITED = config(
     'SUBSCRIPTIONS_WITH_UNLIMITED', default=''
 )
+PREMIUM_RELEASE_DATE = config('PREMIUM_RELEASE_DATE', str(datetime.now(timezone.utc)), cast=str)
```

Comment:
can we go ahead and do `datetime.fromisoformat(config('PREMIUM_RELEASE_DATE', str(datetime.now(timezone.utc)), cast=str))` ? so we can just treat `settings.PREMIUM_RELEASE_DATE` as a `datetime` object elsewhere in the code?

Specifically, I'm thinking of how I'll implement "automatically opted in to data collection" after `settings.PREMIUM_RELEASE_DATE` and I think I'll change `Profile.server_storage` field to:

```
def default_server_storage_value():
    return datetime.now(timezone.utc) > settings.PREMIUM_RELEASE_DATE
...
class Profile(models.Model):
    ...
    server_storage = models.BooleanField(default=default_server_storage_value)
```